### PR TITLE
cpanm: add installdeps option

### PR DESCRIPTION
**Pull request by andskli**
Added the `--installdeps` option to the cpanm module.

### CODE REVIEW

**Comment by resmo** [comment]
LGTM

File: packaging/language/cpanm.py [hunk 1]
```diff
@@ -58,6 +58,11 @@
       - Use the mirror's index file instead of the CPAN Meta DB
     required: false
     default: false
+  installdeps:
+    description:
+      - Only install dependencies
+    required: false
+    default: false
```
Comment (resmo) [inline packaging/language/cpanm.py:65]: add `version_added: "2.0"` here as well

**Comment by andskli** [comment]
@resmo Thanks for the feedback. Added.

**Comment by resmo** [comment]
@andskli the version needs to be quoted: `"2.0"`

**Comment by andskli** [comment]
Oh crap. Should be OK now @resmo.

**Comment by resmo** [comment]
LGTM

**Pull request merged**